bundles/framework/personaldata/Flyout.js

```diff
@@ -61,7 +61,11 @@ Oskari.clazz.define('Oskari.mapframework.bundle.personaldata.Flyout',
             // TODO: move these to correct bundle and use AddTabRequest to add itself to PersonalData
             this.tabsData = {
                 'myviews': Oskari.clazz.create('Oskari.mapframework.bundle.personaldata.MyViewsTab', me.instance),
-                'publishedmaps': Oskari.clazz.create('Oskari.mapframework.bundle.personaldata.PublishedMapsTab', me.instance),
+                'publishedmaps': {
+                    customRender: true,
+                    js: Oskari.clazz.create('Oskari.mapframework.bundle.personaldata.PublishedMapsTab', me.instance),
+                    getTitle: () => Oskari.getMsg('PersonalData', `tabs.publishedmaps.title`)
+                },
```

Comment:
I don't see a good reason to complicate this. You could just add
```
getTitle: () => Oskari.getMsg('PersonalData', `tabs.publishedmaps.title`);
```
for the `PublishedMapsTab` and use the existing tab-render loop:
```suggestion
                'publishedmaps':  Oskari.clazz.create('Oskari.mapframework.bundle.personaldata.PublishedMapsTab', me.instance),
```